src/gui/editorwidgets/core/qgseditorwidgetwrapper.h

```diff
@@ -258,8 +286,18 @@ class GUI_EXPORT QgsEditorWidgetWrapper : public QgsWidgetWrapper
      * to reflect the new value.
      *
      * \param value The new value of the attribute
+     * \deprecated since QGIS 3.10
+     */
+    // TODO Q_DECL_DEPRECATED
+    virtual void setValue( const QVariant &value ) SIP_DEPRECATED;
```

Comment:
Anyone had any idea why actually deprecating this method leads to a compile error:

```/Users/denis/opt/QGIS/src/gui/editorwidgets/core/qgseditorwidgetwrapper.h:292: Error: Not a signal or slot declaration
make[3]: *** [src/gui/editorwidgets/core/moc_qgseditorwidgetwrapper.cpp] Error 1
make[2]: *** [src/gui/CMakeFiles/qgis_gui.dir/all] Error 2
make[1]: *** [src/app/CMakeFiles/QGIS.dir/rule] Error 2
make: *** [QGIS] Error 2```